Multisig/UI/Settings/App Settings/Owner Wallet Management/SelectOwnerAddressView.swift

```diff
@@ -60,11 +66,17 @@ struct SelectOwnerAddressView: View {
                                             derivePrivateKey: true)?.privateKey else { return }
         do {
             try App.shared.keychainService.save(data: pkData, forKey: KeychainKey.ownerPrivateKey.rawValue)
```

Comment:
I think having this code inside the view is not the right place, this should be either in a model or view model.